actionpack/test/controller/mime/respond_to_test.rb

```diff
@@ -619,21 +619,25 @@ def test_invalid_variant
     logger = ActiveSupport::LogSubscriber::TestHelper::MockLogger.new
     old_logger, ActionController::Base.logger = ActionController::Base.logger, logger
 
-    get :variant_with_implicit_rendering, params: { v: :invalid }
-    assert_response :no_content
+    assert_raises ActionController::UnknownFormat do
```

Comment:
These all need to be changed to `ActionController::UnknownFormat` because there is a template defined for this controller action (I could find only 1 template), so this means that an error is raised if the format and/or variant is not recognized.